src/forms/RangeField.js

```diff
@@ -14,6 +15,8 @@ const handle = (props) => {
     ...restProps
   } = props;
 
+  const SliderComponent = SliderComponentGivenValue(value);
```

Comment:
I'm not sure if I'm happy with the SliderComponentGivenValue function. `const SliderComponent = _.isArray(value) ? Range : Slider;` would work just fine no?

Reply:
Hmm you seem to be using it in two places. I still find the naming and usage of the function kinda odd. Especially since it starts with a capital letter.

Reply:
@gl3nn yes, however it is being used in two places, so I modularised the logic.